### ISSUE TITLE
TextField bind value not working

### ISSUE THREAD

**Issue by vallemar** [nativescript-vue/nativescript-vue#1015]
I'm trying to bind a ref with TextField but I can't make it work, I've tried this:
```xml
<TextField  v-model="email" :text="email" hint="Email" class="py-2 ml-2 input "/>
```
and this, and the event is not fired: 
```xml
 <TextField  @textChange="onChangeEmail" :text="email" hint="Email" class="py-2 ml-2 input "/>
```

Note: email is a ref

**Comment by heywhy** [comment]
The `textChange` does get fired on the `TextField` component.

**Comment by vallemar** [comment]
@heywhy I'll try again, thanks!

**Comment by vallemar** [comment]
Yes, textChange working now

**Comment by rigor789** [comment]
Should be fixed by the fix for #1023

**Comment by vallemar** [comment]
@rigor789  Can you release a version with this fix?
